core/main/src/service/apps/delegated_launcher_handler.rs

```diff
@@ -115,68 +119,115 @@ pub struct AppManagerState {
     // This is a map <app_id, app_title>
     app_title: Arc<RwLock<HashMap<String, String>>>,
     app_title_persist_path: String,
+    // This is a map <app_id, app_migrated_state>
+    migrated_apps: Arc<RwLock<HashMap<String, Vec<String>>>>,
```

Comment:
Why do we need to store this list in memory ?

Reply:
Code changes are made as per the discussion with @kpears201. Doing the same here as app_title.